docs/root/configuration/http/http_filters/sxg_filter.rst

```diff
@@ -0,0 +1,71 @@
+
+.. _config_http_filters_sxg:
+
+SXG
+======
+
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.sxg.v3alpha.SXG>`
+* This filter should be configured with the name *envoy.filters.http.sxg*.
```

Comment:
Can you note this is in contrib builds only with a ref link to the contrib docs. If the squash filter docs don't have that can you fix that also? (I know I automated the other docs but I think for these manual docs it is probably missing.)